CHANGELOG

```diff
@@ -1,3 +1,105 @@
+cocos2d-x-3.17 May 2018
+
+[HIGHLIGHT]     Android: support Android Studio 3.0, NDK 16
```

Comment:
It is Android Studio 3.0 or 3.1?

Reply:
Our test team tested the Android project using 3.0, and the default Gradle config is suitable for 3.0. So I think we should tell developers we support 3.0, even though some developers said that the project goes well in 3.01, 3.1 only need update the Gradle config.

Reply:
Sounds good, can mention it in release note.